internal/controller/nemocustomizer_controller_test.go

```diff
@@ -453,8 +476,15 @@ var _ = Describe("NemoCustomizer Controller", func() {
 			}))
 
 			Expect(envVars).To(ContainElement(corev1.EnvVar{
-				Name:  "POSTGRES_DB_DSN",
-				Value: nemoCustomizer.GeneratePostgresConnString(),
+				Name: "POSTGRES_DB_DSN",
```

Comment:
we have to check with ncs team to align on the same env variable name for this.

Reply:
It is `POSTGRES_DB_DSN` in the latest deployment.yaml. Do we need to check with them further?

Reply:
i mean between all microservices, they need to be consistent. currently its POSTGRES_DB_URI and POSTGRES_DB_DSN. we have to suggest them.